config/paths.js

```diff
@@ -46,8 +46,8 @@ module.exports = {
   dotenv: resolveApp('./.env'),
   entry: resolveApp('./src/index.js'),
   appBuild: resolveApp('./build'),
-  elmPackageJson: resolveApp('./elm-package.json'),
-  elmMake: require('elm/platform').executablePaths['elm-make'],
-  publicUrl: getPublicUrl(resolveApp('elm-package.json')),
-  servedPath: getServedPath(resolveApp('elm-package.json'))
+  elmPackageJson: resolveApp('./elm.json'),
+  elm: require('elm/platform').executablePaths['elm'],
```

Comment:
It doesn't look like `executablePaths` is available with the new NPM package. Not sure how best to handle this.

Reply:
First places to check are https://github.com/rtfeldman/node-elm-compiler and https://github.com/elm-community/elm-webpack-loader
I'll be looking into this a bit later today.

Reply:
Yeah node-elm-compiler hasn't been updated recently and my PR to update elm-webpack-loader is https://github.com/elm-community/elm-webpack-loader/pull/142